src/NativeSerializer.js

```diff
@@ -7,6 +7,9 @@ const allowedMapObjectTypes = [ 'string', 'number', 'boolean' ]
 const serializeForNativeLayer = (map, maxDepth = 10, depth = 0, seen = new Set()) => {
   seen.add(map)
   const output = {}
+  if (map instanceof Error) {
```

Comment:
I would use the [iserror](https://github.com/yefremov/iserror)  module for more robust `Error` detection. `instanceof` checks alone can yield [unexpected results](https://groups.google.com/forum/#!msg/comp.lang.javascript/XTWYCOwC96I/70rNoQ3L-xoJ). 

We make similar use of this module in [bugsnag-js](https://github.com/bugsnag/bugsnag-js/blob/9b2309d2a3adedb187dd908e7c4850d228ea2c46/browser/plugins/unhandled-rejection.js#L74).